velox/dwio/parquet/tests/writer/ParquetWriterTestBase.h

```diff
@@ -20,11 +20,10 @@
 #include <string>
 #include "velox/common/base/Fs.h"
 #include "velox/dwio/common/FileSink.h"
-#include "velox/dwio/common/tests/utils/BatchMaker.h"
 #include "velox/dwio/common/tests/utils/DataFiles.h"
 #include "velox/dwio/parquet/writer/Writer.h"
 #include "velox/exec/tests/utils/TempDirectoryPath.h"
```

Comment:
Now that you moved implementation to .cpp file. Do you still need all these includes?

Reply:
Fixed by move some includes into cpp file.